mats/6.ms

```diff
@@ -3332,6 +3332,7 @@
   (error? (path-last 'a/b/c))
   (error? (path-root 'a/b/c))
   (error? (path-extension 'a/b/c))
+  (error? (path-build 'a 'b))
```

Comment:
Add a check for both strings:
```
(error? (path-build 'a "fn"))
(error? (path-build "dir" 'b))
```

Reply:
Thanks, and please remove the (path-build 'a 'b) case.

Reply:
Checks added.